From e8c11c43a9eb8171d150850c498bc75205681ab8 Mon Sep 17 00:00:00 2001
From: Jonas Gorski <jogo@openwrt.org>
Date: Thu, 26 Jun 2014 12:49:33 +0200
Subject: [PATCH 46/48] MIPS: BCM63XX: expand board_info with an of compatible
 string

It will be used for matching board_info structs to a passed dtb until
bcm63xx is fully converted.

Signed-off-by: Jonas Gorski <jogo@openwrt.org>
---
 arch/mips/include/asm/mach-bcm63xx/board_bcm963xx.h | 7 +++++++
 1 file changed, 7 insertions(+)

--- a/arch/mips/include/asm/mach-bcm63xx/board_bcm963xx.h
+++ b/arch/mips/include/asm/mach-bcm63xx/board_bcm963xx.h
@@ -21,6 +21,7 @@
  */
 struct board_info {
 	u8		name[16];
+	u8		of_board_id[32];
 	unsigned int	expected_cpu_id;
 
 	/* enabled feature/device */
